### PR TITLE
Fix download button of PDF viewer in Firefox

### PR DESCRIPTION
**Pull request by nickvergessen**
Downstream of 27521

### CODE REVIEW

File: lib/public/AppFramework/Http/EmptyContentSecurityPolicy.php [hunk 1]
```diff
@@ -399,7 +399,7 @@ public function buildPolicy() {
 
 		if(!empty($this->allowedFrameDomains)) {
 			$policy .= 'frame-src ' . implode(' ', $this->allowedFrameDomains);
-			$policy .= ';';
+			$policy .= ' blob:;';
```
Comment (nickvergessen) [inline lib/public/AppFramework/Http/EmptyContentSecurityPolicy.php]: Should this be `implode(' blob:', $this->allowedFrameDomains)` @LukasReschke ?

**Review by MorrisJobke — APPROVED** [review]
Tested and works 👍

**Review by LukasReschke — COMMENTED** [review]
Not a too big fan of this. Let me see if we can adjust that directly in the pdf.js app to use the proper download endpoint.

**Comment by LukasReschke** [comment]
Replaced by https://github.com/nextcloud/files_pdfviewer/pull/28

**Pull request closed without merging**